samples/Azure.Management.Storage/Generated/BlobContainerCollection.cs

```diff
@@ -56,8 +56,16 @@ internal static void ValidateResourceId(ResourceIdentifier id)
 
         /// <summary>
         /// Creates a new container under the specified account as described by request body. The container resource includes metadata and properties for that container. It does not include a list of the blobs contained by the container. 
-        /// Request Path: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}
-        /// Operation Id: BlobContainers_Create
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName}</description>
```

Comment:
I mean the content:
```suggestion
        /// <term> Request Path </term>
        /// <description> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/default/containers/{containerName} </description>
```

Reply:
I'm not sure why we added the spaces initially. Your preview of the doc looks good to me.